Data/Cleaned/HSE_MWI_ZCTA_Mental_Wellness_Index_Black.csv

```diff
@@ -10943,7 +10943,7 @@
 "34103",73.8684257580513,97.929238780824,64.3146522619851,95.3195570129308,98.4872857761554,92.8865309657861,97.2746396089835,95.9815628286828,80.6854939039562,63.618648960739,44.9894844540442,53.2228141946532,,74.527750292345,4.53057013412282,78.994401931077,42.5113360509321,25.1569822648329,82.7244866687098,45.5959691387183,8.87300812286386,56.0211283234942,66.3367215588797,74.3219194894453,27.6004166794326,35.7042188792549,94.1174667402367,16.1102467214937,95.4263312642927
 "34104",58.5179075971006,81.7460560575892,51.2675710515008,79.8397574707666,77.3216605828126,80.7229474726226,69.4162593577925,79.8196498174844,51.7603881562578,63.9337952270978,33.0813391689877,18.1714394443916,49.7744034707158,74.527750292345,55.0486699860943,78.994401931077,18.5040992992259,25.1569822648329,82.7244866687098,45.5959691387183,8.87300812286386,56.0211283234942,66.3367215588797,53.3597201767305,27.6004166794326,35.7149422470051,94.1174667402367,35.1811513669704,95.0903066224857
 "34105",61.1532448486839,70.9940266503293,40.3781228899392,83.2348573903359,95.3133700426901,91.2825589308915,94.0063725793479,94.9034832642455,80.6854939039562,83.4512124711316,78.4573409878929,80.8747978308439,17.5661605206074,74.527750292345,10.7253397927601,39.3970520260901,2.19392662483397,25.1569822648329,82.7244866687098,45.5959691387183,8.87300812286386,56.0211283234942,66.3367215588797,51.0585419734904,27.6004166794326,35.7149422470051,94.1174667402367,18.1373638586353,96.4250711718859
-"34108",78.3273755930284,97.0010721396845,81.6432385979989,93.5732846625008,98.8724246736373,93.2685763781476,97.9072573160923,96.672956753078,80.6854939039562,83.4512124711316,,80.8747978308439,17.5661605206074,74.527750292345,4.63822724622079,78.994401931077,52.0840012824623,25.1569822648329,82.7244866687098,45.5959691387183,8.87300812286386,56.0211283234942,66.3367215588797,75.4832474226804,21.4804375134042,28.7447532093508,94.1174667402367,27.9173149588798,93.3261772529985
+"34108",78.327543175725,97.0010721396845,81.6432385979989,93.5732846625008,98.8724246736373,93.2685763781476,97.9072573160923,96.672956753078,80.6854939039562,83.4512124711316,,80.8747978308439,17.5661605206074,74.527750292345,4.63822724622079,78.994401931077,52.0840012824623,25.1569822648329,82.7244866687098,45.5959691387183,8.87300812286386,56.0211283234942,66.3367215588797,75.4847815414826,21.4804375134042,28.7447532093508,94.1174667402367,27.9173149588798,93.3261772529985
```

Comment:
Just for record keeping, what led to the value changes for some of the data?

Reply:
I updated one of the scaling factors that was mistyped in Metadata.xlsx (100 should have been 1) and reran the pipeline -- while that doesn't really change any of the rankings in general (and shouldn't) , it looks like one of the rankings changed by numerical error, and doesn't change any interpretations.